packages/composite-checkout/src/components/checkout-steps.js

```diff
@@ -153,25 +157,41 @@ export function CheckoutSteps( { children } ) {
 		totalSteps
 	);
 
-	return steps.map( ( child ) => {
-		stepNumber = nextStepNumber;
-		nextStepNumber = stepNumber === totalSteps ? null : stepNumber + 1;
-		const isStepActive = activeStepNumber === stepNumber;
-		const isStepComplete = !! stepCompleteStatus[ stepNumber ];
-		return (
-			<CheckoutSingleStepDataContext.Provider
-				key={ 'checkout-step-' + stepNumber }
-				value={ {
-					stepNumber,
-					nextStepNumber,
-					isStepActive,
-					isStepComplete,
-				} }
-			>
-				{ child }
-			</CheckoutSingleStepDataContext.Provider>
-		);
-	} );
+	return (
+		<CheckoutStepsWrapperUI
+			className={ joinClasses( [ className, 'checkout__steps-wrapper' ] ) }
+			isLastStepActive={ ! isThereAnotherNumberedStep }
+		>
+			{ nonSteps }
+			{ steps.map( ( child ) => {
+				stepNumber = nextStepNumber;
+				nextStepNumber = stepNumber === totalSteps ? null : stepNumber + 1;
+				const isStepActive = activeStepNumber === stepNumber;
+				const isStepComplete = !! stepCompleteStatus[ stepNumber ];
+				return (
+					<CheckoutSingleStepDataContext.Provider
+						key={ 'checkout-step-' + stepNumber }
+						value={ {
+							stepNumber,
+							nextStepNumber,
+							isStepActive,
+							isStepComplete,
+						} }
+					>
+						{ child }
+					</CheckoutSingleStepDataContext.Provider>
+				);
+			} ) }
+
+			<SubmitButtonWrapperUI isLastStepActive={ ! isThereAnotherNumberedStep }>
+				<CheckoutErrorBoundary
+					errorMessage={ localize( 'There was a problem with the submit button.' ) }
+				>
+					<CheckoutSubmitButton disabled={ isThereAnotherNumberedStep || formStatus !== 'ready' } />
```

Comment:
So because we're moving the submit button into `CheckoutSteps`, that makes `CheckoutSteps` pretty much required for the form. We should note that early in the README somehow; both that it creates the submit button previously included by `Checkout` and that if you don't use it you won't get a submit button. (That actually might not be clear in the current version of the README but I think it's more important now since we have so many "helpful components you can use" which really are not so optional.) Effectively, `CheckoutSteps` has become the form now rather than "a group of steps joined by continue buttons" so we might want to update its description as well.